src/_locales/en_US/messages.json

```diff
@@ -512,5 +512,37 @@
                 "example": "Facebook Like"
             }
         }
+    },
+    "sync_intro": {
+        "message": "Cloud syncing:<ul><li>Requires Firefox/Chrome Sync</li><li>Download combines the lists of sites where your Badger is disabled</li><li>Upload overwrites the list saved in the cloud</li></ul>",
```

Comment:
Just "Cloud sync:" instead of "Cloud syncing:"?

Reply:
The "upload" bullet point should probably come before the "download" bullet point, to match button order.

Reply:
"Upload overwrites ~~the list saved in the~~any existing data in the cloud"? To avoid getting tied to what is specifically saved.

Reply:
Should we change the order of the Import/Download user data buttons above to match the order of the Upload/Download buttons? So it would be "Download" (export) and "Import" (import) user data, and then "Upload" (export) and "Download" (import) cloud data.

Reply:
I would be inclined to change the order of the new buttons, not the old, but will defer to your judgement.

Reply:
Just seems like importing should line up with importing. Not sure whether it's better to have import or export first. Maybe export? I think it's fine to reorder existing UI.

Reply:
While avoiding getting tied to what is saved is _fine_ on the upload bullet, I do think the download bullet should describe current behavior.

Reply:
Agreed.